htcrystalball/__init__.py

```diff
@@ -1,7 +1,8 @@
 from ._version import __version__
 import logging
+from os.path import expanduser, join as opj
 
-SLOTS_CONFIGURATION = "~/.htcrystalball"
+SLOTS_CONFIGURATION = opj(expanduser('~'), '.htcrystalball')
```

Comment:
This entire commit (`Allow config file to be created if it does not exist`) can be squashed into the previous commit (`Change SLOTS_CONFIGURATION target`).